specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -456,7 +455,6 @@
         },
         "confidenceScore": {
           "format": "float",
-          "x-ms-client-name": "confidence",
```

Comment:
Same here, and everywhere "confidence" appears, including "confidenceThreshold".

Reply:
The missing `x-ms-client-name` properties were all added.